src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -194,6 +206,14 @@ def add(
             assert depth.ndim == 2
         if camera_K is not None:
             assert camera_K.ndim == 2
+        if instance_image is None:
+            assert (
+                obs is not None
+            ), "must provide instance image or raw observations with instances"
+            assert (
+                obs.instance is not None
+            ), "must provide instance image in observation if not available otherwise"
+            instance_image = torch.from_numpy(obs.instance)
```

Comment:
Should we have a wrapper method add_obs(obs) that unpacks obs and calls add?

Reply:
done